tests/functional/setups/manu_poromech_nofrac_3d.py

```diff
@@ -473,21 +474,15 @@ def flow_source(self, sd: pp.Grid, time: float) -> np.ndarray:
 
 
 # -----> Geometry
-# Todo: Update after #860 is merged
-class UnitCubeGrid(pp.ModelGeometry):
-    """Class for setting up the geometry of the unit square domain."""
+class UnitCubeGrid:
+    """Class for setting up the geometry of the unit cube domain."""
 
     params: dict
     """Simulation model parameters."""
 
-    fracture_network: pp.FractureNetwork2d
-    """Fracture network. Empty in this case."""
-
     def set_domain(self) -> None:
         """Set fracture network. Unit square with no fractures."""
-        self._domain = pp.Domain(
-            {"xmin": 0, "xmax": 1, "ymin": 0, "ymax": 1, "zmin": 0, "zmax": 1}
-        )
+        self._domain = nd_cube_domain(3, 1.0)
```

Comment:
I notice you use nd_cube_domain(dim, 1). There is also the function unit_cube_domain(dim). Should we remove it? I think there's no reason to have both.